charts/metallb/templates/rbac.yaml

```diff
@@ -188,4 +188,35 @@ roleRef:
 subjects:
 - kind: ServiceAccount
   name: {{ include "metallb.controller.serviceAccountName" . }}
-{{- end -}}
+{{- end }}
+---
+{{- if .Values.prometheus.rbacPrometheus.enabled }}
```

Comment:
if we move this, it must be conditioned also by having either podmonitor OR servicemonitor enabled

Reply:
Done